src/channel-slider/README.md

```diff
@@ -130,4 +136,12 @@ All attributes are reactive:
 | `input` | Fired when the color changes due to user action. |
 | `change` | Fired when the color changes due to user action. |
 | `valuechange` | Fired when the value changes for any reason, and once during initialization. |
-| `colorchange` | Fired when the color changes for any reason, and once during initialization. |
\ No newline at end of file
+| `colorchange` | Fired when the color changes for any reason, and once during initialization. |
+
+### Parts
+
+| Name | Description |
+|------|-------------|
+| `color_slider` | The internal `<color-slider>` element. |
+| `slider` | The exposed [part of the internal `<color-slider>` element](../color-slider/#parts). |
```

Comment:
*Which* exposed part? You want to mention the name.

Reply:
`slider` is the name of the exposed part. Probably, we should distinguish somehow the component's parts and the exposed ones. 🤔